config/initializers/assets.rb

```diff
@@ -26,6 +26,4 @@
 Rails.application.config.assets.precompile += %w( potlatch2/FontLibrary.swf )
 Rails.application.config.assets.precompile += %w( potlatch2/locales/*.swf )
 Rails.application.config.assets.precompile += %w( help/introduction.* )
-Rails.application.config.assets.precompile += %w( iD/img/*.svg iD/img/*.png iD/img/*.gif )
-Rails.application.config.assets.precompile += %w( iD/img/pattern/*.png )
-Rails.application.config.assets.precompile += %w( iD/locales/*.json )
```

Comment:
You're quite right that the live site isn't actually using the asset map... The test site at http://tomh.apis.dev.openstreetmap.org/ which has this branch merged is using it though, which is good!